templates/prelogin/_sections/software_services/shared_pricing_table.html

```diff
@@ -0,0 +1,162 @@
+{# This file alone is available under the BSD 3-Clause license as posted at #}
+{# https://github.com/dimagi/commcare-hq/blob/master/LICENSE #}
```

Comment:
> I think that's okay as long as the shared resource is in HQ, otherwise it'd complicate things for HQ rehosters who don't have legal access to the prelogin submodule

@esoergel Do you think this is enough to get around that?